### ISSUE TITLE
This tool is awesome

### ISSUE THREAD

**Issue by userpixel** [scottnonnenberg/eslint-compare-config#3]
**Big thanks!**

We use it extensively for our config `eslint-config-schibsted` to find differences with other competing configs and improve our compatibility.

👍

**Comment by pebanfield** [comment]
Yes a very cool idea. I've been reviewing the different configs out there and was wondering what it would take to understand the deltas. But hay somebody already did the work. Awesome.
